ui-tests/cypress/support/pageObjects/content/assets/AssetsPage.js

```diff
@@ -49,114 +86,74 @@ export default class AssetsPage extends AdminContent {
   }
 
   getFilterResultInfo() {
-    return this.getAssetsBody()
+    return this.get()
                .find(this.resultInfo);
   }
 
   getFilterResultItemCount() {
     return this.getFilterResultInfo()
+               .find(`#search > .pager`)
                .find(this.resultInfoItemCount);
   }
 
-  getTable() {
-    return this.getAssetsBody()
-               .find(htmlElements.table);
-  }
-
-  getTableRows() {
-    return this.getTable()
-               .children(htmlElements.tbody)
-               .children(htmlElements.tr);
-  }
-
   getKebabMenu(code) {
     return new AssetsKebabMenu(this, code);
   }
 
-  selectFiles(...fileName) {
-    this.getFileInput().selectFile(fileName, {force: true});
-    this.parent.getDialog().setBody(AddAssetDialog);
-  }
 }
 
 class AssetsKebabMenu extends KebabMenu {
 
   get() {
-    return this.parent.getTableRows()
-               .find(`#AssetsList__item-action-${this.code}`)
+    return this.parent.getAssetsBody()
+               .find(`${htmlElements.div}.list-group-item`)
+               .children(`${htmlElements.div}.list-view-pf-actions`)
+               .children(htmlElements.div);
+  }
+
+  getActionsButton() {
+    return this.getDropdown()
```

Comment:
Here I used the KebabMenu method 'getDropdown()'. 
I think this justifies the extension of KebabMenu, despite the overwriting of get() and the uselessness of open() in this context.